### ISSUE TITLE
Database/log of monitor brands and models with issues

### ISSUE THREAD

**Issue by ahydronous** [alin23/Lunar#55]
Would it be interesting to create a log of all monitors that have issues with reading brightness (or other issues) via DCC?

Just the idea of keeping this issue open and having people post their monitor brand and model for easy search, and perhaps even trying to debug/get working those monitor brands (by having people test builds with reduced polling rate or whatever). It would also make it easy to refer people to this issue to check if they have an affected monitor.

## Example

1. Monitor brand & model: AOC Q2778VQE
2. Host device: Macbook Pro 13" 2015 (A1502)
3. Type of connection: Thunderbolt 2 to Displayport via Displayport extension cable
4. Dock: no
5. What works: Setting brightness and contrast both manually, adaptive and location-based
6. What doesn't work: reading the brightness value - macOS completely locks up and even disconnecting the Displayport cable doesn't work, only a reboot via the power button does

**Comment by alin23** [comment]
It's a good idea but I'm afraid people are going to report monitors that should work, but don't work for them because of the way they connect the monitors to their device.

So I think the required fields should be:
1. Monitor brand and model (e.g. AOC Q2778VQE)
2. Host device (e.g. Macbook Pro 2019 15in)
3. Type of connection (e.g. _Thunderbolt 3_ or _HDMI to Thunderbolt 3_ or _DisplayPort to HDMI through Thunderbolt 3 hub_
4. Model of hub/dock if one is used
5. What works (e.g. setting brightness and contrast)
6. What doesn't work (e.g. reading brightness and contrast)

**Comment by ahydronous** [comment]
Sorry for the delay in answering, yes that seems great :). I've edited my initial comment to reflect the updated format

**Comment by jessestauffer** [comment]
1. **Monitor brand and model:** 2 dual LG 27UL600-W
2. **Host device:** Macbook Pro 2019 16in
3. **Type of connection:** HDMI to USB-C
4. **Model of hub/dock:** None
5. **What works:** Changing settings for 1 of the monitors
6. **What doesn't work:** Changing settings for the 2nd monitor

I saw on another issue that sometimes Lunar has trouble identifying individual monitors in a dual monitor setup. Any ideas on how I could get this working?

**Comment by alin23** [comment]
Hi @jessestauffer! 
Does the ddcctl utility work for you for both monitors? You can find it here: https://github.com/kfix/ddcctl

If that works, then we could hop on a private chat to diagnose this issue.

Usually Lunar has trouble with multiple monitors when they are connected through the same port, like through a multiplexing USB-C hub. If the monitors are connected to individual USB-C ports of the Macbook then it shouldn't have any problems _if the OS correctly recognises the monitors and can uniquely identify them_

**Comment by jessestauffer** [comment]
Thanks for the response, @alin23! I tried the ddcctl utility and it recognizes two monitors but only worked on the first monitor, similar to Lunar. I'm waiting on a replacement HDMI to USB-C to the faulty monitor, due to screen lag. I'm curious to see if the faulty cable could be the cause of this issue as well...

**Comment by alin23** [comment]
In this case it sounds like a cable or connector issue. You can try to swap cables between monitors to see if that is really the issue. 

You can also try to connect the monitors on different sides of the Macbook, one on the left and one on the right, if you have 4 USB-C ports. This way you will be using one Thunderbolt chip per monitor.

I had faulty cables, so that is a possibility. But even more often I had OS problems, where macOS can't uniquely identify the monitors because their EDID data is completely identical. So it's best to try everything before replacing the cable.

**Comment by jessestauffer** [comment]
Tried distributing over the Thunderbolt chips and swapping cables...definitely seems like a cable issue. I'll get a replacement and reach back out if I still have an issue.

Thanks for the help!

**Comment by alin23** [comment]
So it seems that this database isn't that useful after all as all the monitors have DDC implemented and the issues are almost always related to a faulty connection. It's impossible to keep a database of no-name HDMI cables and USB-C docks as the number of combinations for the connection types is enormous.
